docs/release_1.2/release_notes_1.2.md

```diff
@@ -0,0 +1,33 @@
+# Avengers Assemble v1.2 Release Notes
+
+With our additions, we are confident that Avengers Assemble has a MVP. Although some features require refinement, we have implemented enough features with sufficient quality to ensure a working product for head TAs for CS1101S.
+## What's New
+### 1. Export Feature
+Avengers Assemble now supports exporting data of up to 500 contacts to a CSV file!
+
+1. Simply type `export` to export a CSV file of the contacts. 
+<img src="export_1.png" alt="Typing in the `export` command.">
+
+2. Once this message shows, the CSV is available for external use by the head TA.
+<img src="export_2.png" alt="The CSV will be exported.">
+
+3. An example of the exported CSV file.
+<img src="export_csv_result.png" alt="The CSV file.">
+
+### 2. Find Feature
+Avengers Assemble now supports finding contacts by their various details, and not just their names!
+
+1. Simply type `find` to find a contact by their details, followed by the field you want to search up. For example, the screenshot below shows finding contacts with an address containing `30`.
+<img src="find_1.png" alt="Typing in the `find` command.">
+2. The result will show all contacts with an address containing `30`.
+<img src="find_2.png" alt="The result of the `find` command.">
+
```

Comment:
Along with @jayllo-c feature can you also add the `import` feature
Here is a draft i written. I have also included the **after_import** screenshot
![after_import](https://github.com/AY2324S2-CS2103T-T10-1/tp/assets/109005735/acdbc579-8ada-4b9f-9a7d-e2b66fce0a40)


### 2. Import Feature
Avengers Assemble now supports importing data of up to 500 contacts stored in a csv file

1. Simply type `import i/<filePath>` where filePath is the absolute path of your csv file
2. Once this message shows, imported contacts will be reflected in addressBook
<img src="after_import.png" alt="Import success">